cli/cli_chestxray_tutorial_test.sh

```diff
@@ -13,42 +13,57 @@ wget -P $DIRECTORY https://storage.googleapis.com/medperf-storage/chestxray_tuto
 tar -xzvf $DIRECTORY/sample_raw_data.tar.gz -C $DIRECTORY
 chmod a+w $DIRECTORY/sample_raw_data
 
+
+# the config does not exists still, but is created automatically by any command
+echo "====================================="
+echo "Changing storage to tmp location"
+echo "====================================="
+# this 'move' command is used only for config updates.
+ev medperf storage move -t $MEDPERF_STORAGE
+checkFailed "moving storage failed"
+echo "We have to return the content back manually, so new tmp folder is empty"
+echo "and all the existing data is kept on the old place."
+mkdir -p ~/.medperf
+ev mv -f $MEDPERF_STORAGE/.medperf/* ~/.medperf/
```

Comment:
a crutch: without it
(1) new tmp storage is spoiled with existing data
(2) it might be deleted after test run, so deleting any files you might want to keep

Reply:
If the intention is to keep test files in case of debugging or similar, I'd prefer to instead specify the location to store them as an argument instead of merging it with the default medperf storage. This is because there's a chance the test files would collide with other files someone might be working on (e.g. I have to do a demonstration of medperf and I'm using the local profile and server for that).

Alternatively, have a flag that disables deleting the tmp folder, and not letting tests run if that folder isn't empty. 

I'd personally prefer tests artifacts to be isolated from the real medperf storage/server if possible.

Reply:
Disregard my message above, I misunderstood this section given the snippet of code commented and the accompanying message. This is only executed when moving the storage failed for some reason right? And it's just to restore the moved files to their original locations. Correct?

Reply:
No:)
`medperf storage move` makes two essential actions:
1. updates all paths in config to point to a new folder
2. actually moves the content of `~/.medperf/` to `/tmp/medperf_tests_XXXYYY/storage/.medperf/`

While (1) is what we want for tests (remember, we have a separate temporary config file while running tests), (2) is eww. Because really we want to work with a new clean empty storage folder while running tests (and, and, in particular, to know it's can be safely deleted at the end).

So, this crutch reverts (2) and moves storage content back to `~/.medperf/`.